desc/plotting.py

```diff
@@ -1656,8 +1655,8 @@ def plot_boundary(eq, phi=None, plot_axis=True, ax=None, return_data=False, **kw
             color=colors[i],
             linestyle=ls[i],
             lw=lw[i],
-            label="$\\phi \\cdot NFP/2\\pi = {:.3f}$".format(
-                grid.NFP * phi[i] / (2 * np.pi)
+            label="$\\phi \\cdot N_{{FP}}/2\\pi = {:.2f}$".format(
```

Comment:
If you are making this change here, might as well make it to all the other plot functions which use NFP in the latex string